src/features/node-fetch.js

```diff
@@ -0,0 +1,4 @@
+import fetch from 'node-fetch';
+
+global.System.constructor.prototype.shouldFetch = () => true;
+global.System.constructor.prototype.fetch = url => fetch(url);
```

Comment:
Should we extend this with `file:///` URL support? See the jspm3 codebase `fetch.ts` file for inspiration here if necessary.

Reply:
👍 added